File: CS/EntityFrameworkCodeFirstBindingExample/Car.cs
namespace EntityFrameworkCodeFirstBindingExample
{
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    public partial class Car
    {
        public int ID { get; set; }

        [StringLength(50)]
        public string Trademark { get; set; }

        [StringLength(50)]
        public string Model { get; set; }

        public short? HP { get; set; }

        public double? Liter { get; set; }

        public byte? Cyl { get; set; }

        public byte? TransmissSpeedCount { get; set; }

        [StringLength(3)]
        public string TransmissAutomatic { get; set; }

        public byte? MPG_City { get; set; }

        public byte? MPG_Highway { get; set; }

        [StringLength(7)]
        public string Category { get; set; }

        public string Description { get; set; }

        [StringLength(50)]
        public string Hyperlink { get; set; }

        [Column(TypeName = "image")]
        public byte[] Picture { get; set; }

        [Column(TypeName = "money")]
        public decimal? Price { get; set; }

        public string RtfContent { get; set; }
    }
}

File: VB/EntityFrameworkCodeFirstBindingExample/Car.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace EntityFrameworkCodeFirstBindingExample

    Public Partial Class Car

        Public Property ID As Integer

        <StringLength(50)>
        Public Property Trademark As String

        <StringLength(50)>
        Public Property Model As String

        Public Property HP As Short?

        Public Property Liter As Double?

        Public Property Cyl As Byte?

        Public Property TransmissSpeedCount As Byte?

        <StringLength(3)>
        Public Property TransmissAutomatic As String

        Public Property MPG_City As Byte?

        Public Property MPG_Highway As Byte?

        <StringLength(7)>
        Public Property Category As String

        Public Property Description As String

        <StringLength(50)>
        Public Property Hyperlink As String

        <Column(TypeName:="image")>
        Public Property Picture As Byte()

        <Column(TypeName:="money")>
        Public Property Price As Decimal?

        Public Property RtfContent As String
    End Class
End Namespace
